### DISCUSSION TITLE
新发布的包2.0.1和2.0无法启动，just quit unexpectedly, please check logs

### DISCUSSION THREAD

**Discussion by zpc870921** [apolloconfig/apollo#4538] [General]
$ ./startup.sh
Windows new JAVA_HOME is: /e/tools/soft/java/jdk
Mon Aug 29 13:04:24     2022 ==== Starting ====
Started [1096]
Waiting for server startup.......
pid - 1096 just quit unexpectedly, please check logs under /opt/logs/100003172 and /tmp for more information!

版本2.0.1和2.0.0都有这个问题，日志也没有，就是启动不了

**Comment by nobodyiam** [comment]
Is there any log under opt/logs/100003172 and /tmp?
